CONTRIBUTING.md

```diff
@@ -45,23 +50,31 @@ Here are the current repos for the gonum project.  If code you want to contribut
 
 ### Your First Code Contribution
 
-Before your first merge, you will need to be added to the [CONTRIBUTORS](https://github.com/gonum/license/blob/master/CONTRIBUTORS) and [AUTHORS](https://github.com/gonum/license/blob/master/AUTHORS) file.  Open a pull request adding yourself to them.
+If you are a new contributor, thank you!  Before your first merge, you will need to be added to the [CONTRIBUTORS](https://github.com/gonum/license/blob/master/CONTRIBUTORS) and [AUTHORS](https://github.com/gonum/license/blob/master/AUTHORS) file.  Open a pull request adding yourself to them.
+
+### Code Contribution
+
+If it is possible to split a large pull into two or more smaller pull requests, please try to do so.  Pulls should include tests for any new code before merging.  It is ok to start a pull request on partially implemented code to get feedback, and see if your approach to a problem is sound.  You don't need have to have tests, or even have code that compiles to open a pull.  Benchmarks are optional for new features, but if you are submitting a pull justified by performance improvment, you will need benchmarks to measure the impact of your change, and the pull should include a report from [benchcmp](https://godoc.org/golang.org/x/tools/cmd/benchcmp).
+
+### Code Review
+
+If you are a contributor, please be welcoming to new contributors.  [Here](http://sarah.thesharps.us/2014/09/01/the-gentle-art-of-patch-review/) is a good guide.  We follow the convention of requiring at least 1 LGTM before a merge.  If a submitter or reviewer wants, they can request additional review from others and more LGTMs before merge, which should be used when the code is tricky or could be controversial.
 
-### Pull Requests
+### What Can I Do to Help?
 
-We follow the convention of requiring at least 1 LGTM (for simple changes) and usually 2+ LGTM before a merge.  If it is possible to split a large pull into two or more smaller pull requests, please try to do so.  Pulls should include tests for any new code.  Benchmarks are optional for new features, but if you are submitting a pull justified by performance improvment, you will need benchmarks to measure the impact of your change, and the pull should include a report from [benchcmp](https://godoc.org/golang.org/x/tools/cmd/benchcmp).
+If you are looking for some way to help the Gonum project, there are good places to start, depending on what you are comfortable with.  You can [search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+user%3Agonum) for open issues in need of resolution.  You can improve documentation, or write longer examples.  You can add and improve tests.  You can improve performance, either by improving accuracy, speed, or both.  You can suggest and implement new features that you think belong in Gonum.
 
 ### Style
```

Comment:
We can just link to [Go style](https://github.com/golang/go/wiki/CodeReviewComments).